### DISCUSSION TITLE
Show number of days instead of "NEW" when age is greater than the stability days requirement

### DISCUSSION THREAD

**Discussion by richardkazuomiller** [renovatebot/renovate#13180] [Request Help]
### What would you like Renovate to be able to do?

The age badge seems to say "NEW" for a long time. I'm not sure how long, but the earliest badge I can find that has a number is 22 days. If the definition of "NEW" is 21 days, I think that's quite long in some cases. Although the stability days PR check tells us whether the PR is new enough or not, it would be nice to be able to see `3d` when the stability days requirement is set to 3 days, for example.

### If you have any ideas on how this should be implemented, please tell us here.

Show "NEW" when the stability days is < a smaller constant number like 3, or if 21 is the magic number for the default, use `stabilityDays` as the point where packages are no longer shown as "new" in the badge.

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
NEW is also used when the package/release is not found in the DB. Is it a private package?

**Comment by richardkazuomiller** [comment]
No, I'm seeing it for stuff like eslint

**Comment by richardkazuomiller** [comment]
Here's the URL of the badge: <https://badges.renovateapi.com/packages/npm/eslint/8.4.1/age-slim>

It's 11 days old right now

**Comment by rarkins** [comment]
This was a problem in the app's background processing and not any bug directly on this project. It's now resolved

**Comment by richardkazuomiller** [comment]
Oh great, thank you!
